File: Seccion 11 - Funciones en C++/Seccion_11_Dart/21_parametros_estructura.dart

import 'dart:io';

class Persona {
  String nombre;
  int edad;

  Persona();
}

Persona p1 = new Persona();

void pedirDatos(Persona p) {
  stdout.writeln('Digite su nombre: ');
  p.nombre = stdin.readLineSync();
  stdout.writeln('Digite su edad: ');
  p.edad = int.parse(stdin.readLineSync());
}

void mostrarDatos(Persona p) {
  print('\n\nNombre: ${p.nombre}');
  print('Edad: ${p.edad}');
}

void main() {

  pedirDatos(p1);
  mostrarDatos(p1);
  
}

File: Seccion 11 - Funciones en C++/Seccion_11_Python/21_parametros_estructura.py

class Persona:
    nombre = ''
    edad = 0
    
    def __init__(self):
        pass

p1 = Persona()

def pedir_datos():
    global p1
    
    p1.nombre = input('Digite su nombre: ')
    p1.edad = int(input('Digite su edad: '))

def mostrar_datos(persona):
    print('\n\nNombre: {}'.format(persona.nombre))
    print('Edad: {}'.format(persona.edad))

pedir_datos()
mostrar_datos(p1)